### PR TITLE
[Docs] Added Hindi translation (partial)

### PR DESCRIPTION
**Pull request by arimanyus**
I've added Hindi as a new language. For now, README and summary have been added in Hindi. These are my custom translations. Required entries have also been made in index.html as well as _lans.md

## Description

I have translated the README and Summary in Hindi for now. I'll continue the translations and add more over the course of time. Entries have been made in index and lang.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by D3v3sh5ingh** [comment]
can i take up this forward and review/add more?
Is this still open?

File: docs/hi/faq_general.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+एक तकनीकी दृष्टिकोण से TMK कई नई सुविधाओं को जोड़कर QMK बनाता है। विशेष रूप से QMK ने उपलब्ध कीकोड की संख्या का विस्तार किया है और इनका उपयोग उन्नत सुविधाओं को लागू करने के लिए किया, जैसे `S()`, `LCTL()`, एवम् `MO()`. आप इन क[ीकोड़](keycodes.md) की पूरी सूची देख सकते हैं।
+
+एक परियोजना और सामुदायिक प्रबंधन के दृष्टिकोण से TMK सामुदायिक समर्थन के साथ, सभी आधिकारिक तौर पर समर्थित कीबोर्ड को खुद बनाए रखता है. अलग-अलग गुठ के फोर्क्स मौजूद हैं और अन्य कीबोर्ड के लिए बनाये जा सकते है। डिफ़ॉल्ट रूप से केवल कुछ कीमैप प्रदान किए जाते हैं, इसलिए उपयोगकर्ता आमतौर पर एक दूसरे के साथ कीमैप को साझा नहीं करते हैं। QMK गुणवत्ता मानकों का पालन करने वाले सभी Pull Requests को स्वीकार करते हुए, एक केंद्रित प्रबंधित रिपॉजिटरी के माध्यम से कीबोर्ड और कीमैप दोनों को साझा करने के लिए प्रोत्साहित करता है। यह ज्यादातर समूहो द्वारा संभाले जाते हैं, लेकिन QMK टीम आवश्यक होने पर भी मदद करती है।
```
Comment (D3v3sh5ingh) [inline docs/hi/faq_general.md]: Please check following spellings
गुट
फोर्क
 हैं. not है
सँभाल

**Comment by D3v3sh5ingh** [comment]
Add hacktoberfest Label

**Comment by rishabhx64** [comment]
can i take up this forward and review/add more?
Is this still open?

File: docs/index.html [hunk 2]
```diff
@@ -39,11 +39,11 @@
       search: {
         paths: 'auto',
         placeholder: {
-          '/zh-cn/': '搜索',
+          '/zh-cn/': '搜索', '/hi/': 'खोजें',
           '/': 'Search'
         },
         noData: {
-          '/zh-cn/': '没有结果!',
+          '/zh-cn/': '没有结果!', '/hi/': 'कोई परिणाम नहीं',
```
Comment (yanfali) [inline docs/index.html]: please change this country code to the correct one for india and hindi

File: docs/index.html [hunk 3]
```diff
@@ -39,11 +39,11 @@
       search: {
         paths: 'auto',
         placeholder: {
-          '/zh-cn/': '搜索',
+          '/zh-cn/': '搜索', '/hi/': 'खोजें',
```
Comment (yanfali) [inline docs/index.html]: see next comment

File: docs/index.html (hunk 2 above)
Comment (yanfali) [reply docs/index.html]: actually on 2nd thoughts. Add another entry, don't replace the Chinese one please.

Comment (yanfali) [reply docs/index.html]: ok, please format this correctly. I mistakenly thought it wasn't set up correctly. Add a CR

File: docs/hi/faq_general.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+[QMK](https://github.com/qmk), क्वांटम मैकेनिकल कीबोर्ड के लिए संक्षिप्त, कस्टम कीबोर्ड के लिए उपकरण बनाने वाले लोगों का एक समूह है। हमने [QMK फर्मवेयर](https://github.com/qmk/qmk_firmware) के साथ शुरुआत की थी, जो [TMK](https://github.com/tmk/tmk_keyboard) की एक भारी संशोधित शाखा है।
+
+### क्वांटम नाम हीं क्यों?
+
+<!-- FIXME -->
+
+## QMK और TMK के बीच क्या अंतर हैं?
+
+TMK को मूल रूप से [Jun Wako](https://github.com/tmk) द्वारा डिजाइन और कार्यान्वित किया गया था। जबकि QMK [Jack Humbert](https://github.com/jackhumbert) के फोर्क से शुरू हुआ जो प्लैंक के लिए किया गया था‍‍‍‍‍‍। कुछ समय के बाद जैक का फोर्क TMK से काफी अलग हो गया, और 2015 में जैक ने अपने फोर्क का नाम बदलकर QKK करने का फैसला किया।
```
Comment (MetalDent) [inline docs/hi/faq_general.md]: In this `जैक` is a name so I think you can write it in English (Jack).

**Comment by tzarc** [comment]
At minimum, there are conflicts here that would need to be changed.

This PR hasn't seen progress in quite some time. As such, I'm removing the labels that prevent stalebot from auto-closing this PR. You're free to add commits or comments to keep it alive, but for now there's no reason to keep it open if it's going to sit here inactive.

The requirements for PRs have changed since it was raised, so if you'd like to continue the process of getting this into QMK, please refer to the [PR Checklist](https://docs.qmk.fm/#/pr_checklist) and make the required changes in your next batch of commits. Feel free to reach out to any of the collaborators if you're unsure of things, or even better, ping one of the collaborators on [Discord](https://discord.gg/Uq7gcHh) and we can take it from there. Thanks!

**Pull request closed without merging**